### PR TITLE
[bitnami/openldap] New option LDAP_CUSTOM_SCHEMA_DIR

### PR DESCRIPTION
**Pull request by tdevetzis**
Allow for multiple custom schemas on initialization.

### CODE REVIEW

File: bitnami/openldap/2.6/debian-11/rootfs/opt/bitnami/scripts/libopenldap.sh [hunk 1]
```diff
@@ -367,6 +368,23 @@ ldap_add_custom_schema() {
...
+# Arguments:
+#   None
+# Returns
+#   None
+#########################
+ldap_add_custom_schemas() {
+    info "Adding custom Schemas : $LDAP_CUSTOM_SCHEMA_DIR ..."
+    find "$LDAP_CUSTOM_SCHEMA_DIR" -maxdepth 1 \( -type f -o -type l \) -iname '*.ldif' -print0 | sort -z | xargs --null -I{} bash -c ". /opt/bitnami/scripts/libos.sh && debug_execute slapadd -F \"$LDAP_ONLINE_CONF_DIR\" -n 0 -l {}"    
+    ldap_stop
+     while is_ldap_running; do sleep 1; done
```
Comment (corico44) [inline bitnami/openldap/2.6/debian-11/rootfs/opt/bitnami/scripts/libopenldap.sh:384]: Can you fix the indentation, please?

File: bitnami/openldap/2.6/debian-11/rootfs/opt/bitnami/scripts/libopenldap.sh [hunk 2]
```diff
@@ -506,6 +524,9 @@ ldap_initialize() {
         if [[ -f "$LDAP_CUSTOM_SCHEMA_FILE" ]]; then
             ldap_add_custom_schema
         fi
+        if ! is_dir_empty "$LDAP_CUSTOM_SCHEMA_DIR"; then
+            ldap_add_custom_schemas
```
Comment (corico44) [inline bitnami/openldap/2.6/debian-11/rootfs/opt/bitnami/scripts/libopenldap.sh:528]: Can you fix the indentation, please?

File: bitnami/openldap/2.6/debian-11/rootfs/opt/bitnami/scripts/libopenldap.sh [hunk 3]
```diff
@@ -367,6 +368,23 @@ ldap_add_custom_schema() {
...
+# Add custom schemas
+# Globals:
+#   LDAP_*
+# Arguments:
+#   None
+# Returns
+#   None
+#########################
+ldap_add_custom_schemas() {
+    info "Adding custom Schemas : $LDAP_CUSTOM_SCHEMA_DIR ..."
```
[the remark this answers is not in the document]
Comment (tdevetzis) [reply bitnami/openldap/2.6/debian-11/rootfs/opt/bitnami/scripts/libopenldap.sh]: Done.

File: bitnami/openldap/2.6/debian-11/rootfs/opt/bitnami/scripts/libopenldap.sh (hunk 1 above)
Comment (tdevetzis) [reply bitnami/openldap/2.6/debian-11/rootfs/opt/bitnami/scripts/libopenldap.sh:384]: Can you elaborate a bit please? The indentation seems consistent with the surrounding code.

File: bitnami/openldap/2.6/debian-11/rootfs/opt/bitnami/scripts/libopenldap.sh (hunk 2 above)
Comment (tdevetzis) [reply bitnami/openldap/2.6/debian-11/rootfs/opt/bitnami/scripts/libopenldap.sh:528]: As above.

Comment (corico44) [reply bitnami/openldap/2.6/debian-11/rootfs/opt/bitnami/scripts/libopenldap.sh:528]: I've checked it again and it's already in the correct format.

File: bitnami/openldap/2.6/debian-11/rootfs/opt/bitnami/scripts/libopenldap.sh (hunk 1 above)
Comment (corico44) [reply bitnami/openldap/2.6/debian-11/rootfs/opt/bitnami/scripts/libopenldap.sh:384]: Line 384 should have one less space to have a correct format.

Comment (tdevetzis) [reply bitnami/openldap/2.6/debian-11/rootfs/opt/bitnami/scripts/libopenldap.sh]: All set.

**Review by corico44 — APPROVED** [review]
LGTM

**Pull request merged**